app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -156,13 +156,15 @@ def index
       end
 
       def_param_group :callback do
-        param_group :callback_target
+        param_group :callback_target, TasksController
```

Comment:
Does this really need to be specified? According to the docs the lookup should be done within the current controller

Reply:
Yes it does, when used inside definition of `Array` block: I've found the hard way